scanner/src/main/java/edu/ucr/cs/riple/scanner/out/MethodInfo.java

```diff
@@ -157,4 +159,15 @@ private String getVisibilityOfMethod() {
     }
     return "package";
   }
+
+  /**
+   * Calculates hash. This method is used outside this class to calculate the expected hash based on
+   * instance's properties value if the actual instance is not available.
```

Comment:
This doc could be made a bit clearer, perhaps by pointing out that the hash is calculated based on a `MethodSymbol` instance since we want to predict the hash of a potential `MethodInfo` object without creating the `MethodInfo` instance?

Reply:
806f60d9bdb8d2793fe9525312a3c0e4f78ef22e